src/AspNetCoreModuleV2/AspNetCore/dllmain.cpp

```diff
@@ -40,6 +41,7 @@ BOOL WINAPI DllMain(HMODULE hModule,
     switch (ul_reason_for_call)
     {
     case DLL_PROCESS_ATTACH:
+        g_hServerModule = hModule;
```

Comment:
Quite a long way to propagate the HINSTANCE. Any other way to get the module?

Reply:
This is the most reliable way to do it.